django/conf/urls/__init__.py

```diff
@@ -10,4 +13,9 @@
 
 
 def url(regex, view, kwargs=None, name=None):
+    warn(
+        'django.conf.urls.url is deprecated in favour of django.urls.re_path',
+        RemovedInDjango40Warning,
+        stacklevel=2
```

Comment:
Why `stacklevel=2`? :thinking: IMO we can remove it.

Reply:
I added `stacklevel=2` back in #12960. IMO, it assists the library user with identifying the line they should modify. Without the argument, the warning points to Django code, but with the argument, the warning points to the user's code that much change. There is more detail in the PR too.

Reply:
True, thanks :+1: